tests/stdlib/tstring.nim

```diff
@@ -0,0 +1,52 @@
+discard """
+  file: "tstring.nim"
+  output: '''
```

Comment:
Will change this. I just find it convenient when you need to reset state for each test.